docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,306 @@
+| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain in the `TRIGGER_1` and `TRIGGER_2` states. <br> Default: `false` |
+
+Note that `ALARM` is by default silent and only a label overlay, but you can [poll the classifications](/ml/vision/#getclassificationsfromcamera) to set up a physical alarm trigger.
+
+## Configure a transform camera
+
+To view the classifications that your verification system makes, configure a transform camera.
+
+Navigate to the **Config** tab of your machine’s page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**. Select the `camera` type, then select the `transform` model. Enter a name for your camera and click **Create**.
+
+With **Builder** mode selected, copy and paste the following attributes JSON into your camera's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pipeline": [
+    {
+      "type": "classifications",
+      "attributes": {
+        "classifier_name": "security",
+        "confidence_threshold": 0.5
+      }
+    }
+  ],
+  "source": "my-webcam"
+}
+```
+
+Note that the camera name in `"source"` and the `"classifier_name"` match the names that we configured for our source camera and verification system service, respectively.
+If you used different names, edit as applicable.
+View these [instructions to configure a transform camera](/components/camera/transform/) for reference.
+Save your config.
+
+## View your verification system in action
+
+Navigate to the **Control** tab.
+Expand the card matching the name of the transform camera you configured to view an image stream with the system's notifications overlaid:
+
+![Verification camera feed](/tutorials/verification-system/disarmed.png)
+
+## Next Steps
+
+Now that you've got the verification aspect of your system working, you can use this as a launch point for customizing your own DIY home security system.
+For example:
+
+- Try making it so your machine triggers a physical alarm when the `ALARM` classification is emitted. Write a program with a [Viam SDK](/sdks/) and use [`GetClassificationsFromCamera()`](/ml/vision/#getclassificationsfromcamera) to poll for the classifications made.
+- Try changing the type of [detectors](/ml/vision/detection/), using different detectors for the `TRIGGER_1` and `TRIGGER_2` states.
+- Configure the [filtered camera module](https://app.viam.com/module/erh/filtered-camera) and use it as the source camera in your verification system to save images to the Viam cloud when the system enters into specific states.
```

Comment:
Opt: we could link to our tutorial on it instead, which itself links to the module: https://docs.viam.com/tutorials/projects/filtered-camera/. Up to you.

But definitely mention that only matching images are sent, which is the primary selling point of that module. Maybe:


```suggestion
- Add the [filtered camera module](/tutorials/projects/filtered-camera/) to your machine, and use it as the source camera in your verification system in order to save images to the Viam cloud only when the system enters into specific states.
   This way, you could limit the images captured and synced to only those you are interested in reviewing later, for example.
```

Reply:
Haha when this was written there was no tutorial, sounds good!